### DISCUSSION TITLE
README: current status of LG webOS 5 support (CX, GX, ...)

### DISCUSSION THREAD

**Discussion by PredatH0r** [PredatH0r/ChanSort#207] [General]
updated Jan 08, 2021

**Reordering only works under special conditions**
After receiving numerous positive reports of users successfully importing a modified list, I think it is safe to say now, that the import works under similar restrictions that applied to all LG software versions before webOS 5.
1) Install the latest LG software (either over-the-air, or via internet-connected TV or download manually from LG and install via USB)
2) Channel search with no provider / satetellite selected, "blind scan" selected, "network scan" optional (maybe should be selected)
3) recommendation: use a quality brand USB stick <= 16GB, formatted with FAT32 (not NTFS, nor exFAT32).

**Favorite lists are not supported**
Starting with webOS 5, the favorite lists are not part of the exported data, which makes it impossible to edit favorites with ChanSort. 

**1) TV software version / webOS 5**
There has been confusion about the software version in the TV as it shows different numbers in different menu screens
- Support / Software-Update / Software version: something similar to 3.21.10 
- Support / TV-Information / webOS TV-Version: something similar to 5.2.0-2508
It is possible (but unconfirmed) that a certain minimum version including bugfixes is needed for a successful import.
There were reports about successful imports with software versions 3.11.05, 3.21.9 and 3.21.10.

**2) Clean channel search**
When setting up the TV or running a channel search, you MUST NOT select a specific cable or satellite provider (always use "None", "Other" or similar, even if your provider is listed). 
You also MUST select "Blind scan". 
"Network search" is optional, I suggest to select it.
You should end up with a list that is only sorted by frequency/transponder without a useful logical order in the program numbers.
**If your list has for example "Das Erste HD" on program number 1 (and mybe 100 and 1000 too), your list is presorted, Astra 19.2E Germany list and cannot be imported back into the TV.**
Personally I don't possess a webOS 5 TV and can't run tests myself. So everything I say here is based on experience with previous LG software versions and feedback from users.   

**3) USB-Stick**
NTFS and exFAT32 are proprietary Microsoft file systems, made for Windows. TVs however run on Linux based operating systems and sometimes fail to write or read data correctly from a stick with those file systems. FAT32 (without the "ex") is fully documented and is guaranteed to work under Windows and Linux.
Also use a quality brand stick if you can, preferrably <= 16GB. Some noname thumb drives have defective memory cells or report more memory than they actually have, resulting in data corruption.

**What can go wrong**
- TV still shows the same numbers from before the import. Sometimes importing the same file again helps
- TV shows all programs as 0 and/or reshuffles programs more or less randomly
- TV is unable to import the unmodified backup file you originally exported and kept in a save place

In the worst case you can reset your TV to factory defaults and either run a new channel search or skip the search with the back/exit button on the remote and import a backup from USB.

**Recommendations**
- export your current list, rename it and keep it save. You can use it as a fall-back or a reference list for ChanSort later.
- check for firmware updates. Updates may fix bugs in the TVs import function.
- after a software update the old data stored in the TV may not be 100% compatible with the new software. Usually it is not needed to execute a factory reset after an update, but it's the only guaranteed way to start with a clean slate.

**Feedback**
In any case I very much appreciate any feedback about both successful and failed imports.  
If possible, please provide information about:
- the model number, software and webOS versions of your TV
- the GlobalClone00001.TLL (=edited) and .TLL.bak (=original) files
- steps describing what you did

Regards,
Horst Beham (ChanSort author)

**Comment by kongo09** [comment]
I can report back that I managed to export the channels, change the channel numbering for all the radio stations and successfully load it back into the device. I have an OLED65GX9LA with software version 03.11.20 with webOS version 5.1.1-3707. However, I did loose all favorites, but actually, I didn't see the favorits already when opening the GlobalClone00001.TLL.

**Reply by PredatH0r** [reply]
Thanks a lot for your feedback!

So far I have been unable to find out if there even is any information about favorites in the webOS 5 file.
Could you send me your GlobalClone file and let me know a couple example channels that you have in your favorites? 
Does the TV offer a single Fav list or are there more to choose from (A-F, 1-8 or similar)?
It would be ideal if you could export a file, rename it (e.g. to GlobalClone00001.tll.ori), then add a channel to a fav list and run another export. I can then compare the 2 files and find out how the favorites are stored there (if at all).

**Comment by Fallemo** [comment]
Hello.

I tried the following:

My TV: **LG 55UN71003LB**
Software version: **03.11.05**
webOS version: **5.1.0-3203 (jhericurl-japoon)**

1. Searched for cable, no operator (others), full search, blind scan, digital only;
2. Exported the channles on usb drive, ntfs, got my GlobalClone00001.tll;
3. Imported in ChanSort_2021-01-02;
4. File -> Open TV Data -> Create new list -> Add All to left;
5. Used CTRL-K + CTRL-H on every unwanted channel;
6. Ordered by switching the number, or dragging the channel in the list;
7. Select All -> Edit / Renumber channels;
8. Saved the reference txt on my desktop;
9. Saved the file and appended unsorted channels;
10. Copy on the usb drive and imported on the tv.

Now, every channel looks about right, where it is supposed to be and everything seems to be in order.

It's strange, because i tried endlessly for the last week, and when i was browsing through the channels, they would randomly get scrambled.
The only extra step i did this time, was step 1.
Before, i selected an operator from the list (my operator).
So, probably this is the important step!

Thank you for this amazing app.

**Comment by M0squit0** [comment]
Hi,

Current version of Chansort worked without any issues on LG OLED65CX3LA.
TV SW version: 03.11.30
webOS version: 5.1.1-3707

Ran a channel search on tv, exported to usb, loaded into ChanSort, ordered of my wantings, renamed original file list and saved the new on the same name. Inserted pendrive to tv and loaded back. All fine as always in several tvs in last few years I'm using ChanSort :)

Thanks PredatH0r for your work!

**Reply by qrti** [reply]
-> M0squit0
I assume you loaded the list back after a firmware reset? according to the last version log there where no changes referring webOS 5x

my try of the latest version **without** firmware reset ended with a messed up channel list

anyway, thank you PredatH0r for your work, hope dies last ;) ...

LG OLED 48CX 03.11.30 
webOS 5.1.1-3707

**Reply by M0squit0** [reply]
I just bought the tv recently, updated frimware before even starting to use it, ran a channel search and the rest is listed above so never tried on any older firmwares but I'm glad it works now :)

**Reply by Fallemo** [reply]
> -> M0squit0
> I assume you loaded the list back after a firmware reset? according to the last version log there where no changes referring webOS 5x
> 
> my try of the latest version **without** firmware reset ended with a messed up channel list
> 
> anyway, thank you PredatH0r for your work, hope dies last ;) ...
> 
> LG OLED 48CX 03.11.30
> webOS 5.1.1-3707

Check my above steps. I also got a messed up channel list after importing and browsing a little bit.

**Comment by PredatH0r** [comment]
The most important point is that a predefined list cannot be reordered. Normally, when you set up the TV and select something like a specific cable provider or the "Astra 19.2 Liste", you get a preordered list for your country.
So if for example "Das Erste HD" is program 1 (and maybe additionally on 100 and 1000), you have a predefined list and the TV goes nuts when loading any other list from USB (regardless if they were modified with ChanSort or downloaded from some website).

Another possible cause (and I'm just guessing here) is that when you manually changed a program number through the TV's menu, it does set a "user modified" flag in the list. And from what I've seen in the past, I think it tries to keep that program on that number even when you import a list, which may cause conflicts and the weird behavior.

So it's always best to run a full blindscan channel search first, which will typically result in a list with useless order, but allows to be reordered. You can then use another channel list (e.g. from websites like sender-liste.de or others), use that as a reference list for an initial order and continue editing your list afterwards.

**Reply by M0squit0** [reply]
Yes, one thing I have not highlighted is that I can chose from DIGI and Vodafone as cable service provider but even though I have Vodafone service (former UPC) I've chosen "Other Operators" for the scan.

**Comment by qrti** [comment]
this is what I did

full blindscan channel search, export to usb stick, loaded list in chansort, modified it with a predefined list resulting in about 90 useful channels numbered 1..90, when saving the list chansort appended the whole unsorted rest

after transferring the list to tv, my 90 channels first showed up all as number 0, after a while this changed and they appeared at positions at 1800 upwards, order completely destroyed, tv own program sort said '90 active channels', chansort appended channels were grayed out at positions after about 1900

now I started sorting manually on tv, strange enough what happened, lets say I placed 'Das Erste HD', then miraculously the 90 active channels got expanded with all 'Das Erste' found in the appended and greyed out list

after sorting all 90 channels the tv displayed about 500 active channels

it may also be note worthy that I could not enter program numbers greater than 99 with tv program sort

I have an correct list now but am afraid of exporting and importing it ever again ...

**Reply by PredatH0r** [reply]
Could you send me your GlobalClone00001.tll and GlobalClone00001.tll.bak files please? (horst at beham dot biz)
I'm curious if there is anything out of the ordinary in the original file (.bak) or modified file (.tll)

**Comment by qrti** [comment]
here they are:

01 original from TV
02 modfied by chansort
03 corrected 02 on TV
predefined list

[lists qrti.zip](https://github.com/PredatH0r/ChanSort/files/5781739/lists.qrti.zip)

**Reply by PredatH0r** [reply]
The "original" list has "Das Erste HD" on channel 1, 100 and 1000 and is clearly a predefined Astra 19.2E Germany list.
When such a list is in the TV's memory, it is unable to load any other lists from USB.
Normally the TV also blocks reordering in its menu. Maybe that got unlocked by the import.

**Comment by qrti** [comment]
thank you for the hint, when I find time & courage I'll give it an other try ...

**Comment by qrti** [comment]
**hurray**, this time it worked! the only difference was that I checked network **+** blind option for scanning
[https://github.com/PredatH0r/ChanSort/search?q=network&type=wikis](https://github.com/PredatH0r/ChanSort/search?q=network&type=wikis)

the other change was a new firmware and webOS version
[https://www.lg.com/uk/support/product/lg-OLED48CX6LB](https://www.lg.com/uk/support/product/lg-OLED48CX6LB)

**Comment by daabm** [comment]
Worked like a charm after I finally discovered this page and the point about presorted lists/selecting no provider and doing a blind scan. I reordered the export with the predefined Astra Free template, imported to TV. The TV did a reboot after importing and then took its time to populate channel logos and timeline, but no technical problem at all with anything.

**LG 55CX9LA, Web OS 5.2.0-290904, SW Version 3.21.16**

**To me it seems that ChanSort Web OS 5 support is far beyond "experimental" :-))**

**Comment by chio07** [comment]
Thank you for the brilliant program. Here´s my experience with my LG oled55cx9la:

- I accidentally selected a specific cable provider. After exporting the channels I sorted them with chansort and loaded them back on the TV. Everything went well, but on the next day the channels were unsorted again (but I had only the channels I selected in chansort, just in a weird order). I tried a few times, but didn’t manage to find out what’s triggering the reordering of the channels.

- I did a factory reset and a new scan with "network scan" enabled. Unfortunately, with the same result.

- So I uploaded the same file again and swapped two channels of the higher numbers manually. Since that everything is stable!

It seems that the manual correction made the tv not to change the order after the upload.

[GlobalClone00001.zip](https://github.com/PredatH0r/ChanSort/files/6084704/GlobalClone00001.zip)

**Comment by ai-bits** [comment]
**Man, thank you for continuing to dig into this!!! GREAT!**

So, after much boasting in the [Not possible to Edit Channels on new LG CX Series](https://github.com/PredatH0r/ChanSort/issues/172) thread, finally a contribution:

Due to some transponder changes I took the plunge today and, amongst an Orsay 2016 UE55HU7580 and a 2020 Q80T, I re-did / edited the LG CX (03.21.16 / 5.2.0-290904) channel list.

With Anderer Anbieter and Blindsuche the import worked - mostly.

Interestingly the VERY local Austrian LT1-OOE and TV1 OOE are in the list on the PC, but gone on the TV, including Welt der Wunder. (no explanation for the latter either)

Aside: Did a total rescan on the Q80T and seem to remember that Das Erste was missing on position 6 where I had it on the template. More tomorrow.

Thanks for any further insight.
G.

**Reply by ai-bits** [reply]
Hi!
As Anderer Anbieter and Blindsuche didn´t do the trick for the 3 channels (whether they were in the template or added them manually), I did a new total scan adding Netzwerksuche to the mix, but that didn´t do it either. (ChanSort 2021-03-17)

[GlobalClone00001.zip](https://github.com/PredatH0r/ChanSort/files/6188812/GlobalClone00001.zip)
The 3 channels are at the very end. (1288ff or so)

TIA
G.

**Comment by PredatH0r** [comment]
Can you also send me your GlobalClone00001.TLL.bak? ChanSort creates that file as a back up the first time you save a modified .TLL.
I can try and look if there are any out-of-the-ordinary changes for these 3 channels.
They are all on the same transponder (12662 MHz), but there are also other channels on it which seem to work fine for you (ORF III, ServusTV Deutschland and Austria, ...). And also "sixx Austria ALT", which has moved to transponder 11670 recently. 

About missing "Das Erste" on your Samsung TV: would also be nice to get the .zip, .zip.bak and your reference list file.
In the last updates for ChanSort the way reference lists are handled changed a bit. Some channel lists contain the same channel multiple times, maybe that's what causing the problem here (e.g. "Das Erste" on 1, 100 and 1000)

**Reply by ai-bits** [reply]
Ok, I recreated the first variant. (had done some tweaks in the meantime - to no avail)
ZIP contains .TLL, .bak and the .txt template file I derived the list from. The 3 vanishing channels are at 21, 26 and 40 in the template.

Also please mind the channels at the end (1288,..) with ampersands in their name which seem to have fallen victim.

Even Quickstart+ is disabled for a clean boot (required in dev-mode).

Samsung files in the evening.
Cheers
G.
[GlobalClone00001-AndererAnbieter-Blindsuche-Netzwerk.zip](https://github.com/PredatH0r/ChanSort/files/6189788/GlobalClone00001-AndererAnbieter-Blindsuche-Netzwerk.zip)

**Reply by ai-bits** [reply]
Victory, Horst, I got a temporary solution, which is not too clumsy:
While I have still no idea why the LG swallows the 3 channels from the ChanSort list
Sender > Programmsuche und Einstellungen > Sendereinstellung > Manuelle Einstellung > Aktualisierung of the D-TV channels / transponders brings back the swallowed channels in hidden state, _including the ones with an ampersand in the name._

[GlobalClone00001-20210324-2143-after-manual-search-hidden-chans.zip](https://github.com/PredatH0r/ChanSort/files/6200671/GlobalClone00001-20210324-2143-after-manual-search-hidden-chans.zip)

I set the 12662 MHz transponder as target (and checked Network), but it scanned all 99 transponders.

Not too much work then to un-hide in Sender > Sendermanager > Überspringen / Wiederherstellen

Still hoping you find the real cause for the disappearances.
I checked the ChanSort generated file and there were no Skip flags for the cannels in question.
Cheers
G.
Aside: I always need to throw away the config.xml, otherwise ChanSort doubles column widths with every start. Guess someone has experienced this as well and you have a solution for this nuisance!?

Aside 2: When I save a Samsung list ChanSort lets me throw away the Unsorted channels, but not with an LG list!

**Comment by ai-bits** [comment]
So, I just spent a quarter hour scratching my head that Das Erste is not in yesterday´s raw Samsung QE49Q80T channel list.
No wonder that it was missing after template application.
Scrolling through the list it is the ONLY major channel missing. All the others are there: All ARD 3rd prgs, ZDF,..

Actually it was the 2nd full wiping scan with std parameters yesterday, but Senderinitialisierung Aktivieren.
Like the old saying: What can go wrong goes wrong.

Just did the 3rd wiping scan and now it´s there at # 19 where ZDF was before. (now at 20)

Worked like a charm today with the same template file as on / from the LG. Include the files anyway for comparison.
Cheers
G.
[Channel_list_T-NKMDEUC-1497.0-set.zip](https://github.com/PredatH0r/ChanSort/files/6192915/Channel_list_T-NKMDEUC-1497.0-set.zip)

**Comment by PredatH0r** [comment]
Hallo Günter,

 

Sorry dass das so lange gedauert hat.

Leider ist es ziemlich kompliziert bei den vielen Dateien einen Überblick zu bekommen und nachzuvollziehen, was da wie abgelaufen ist. 

Ich vermute, dass du eine alte, bestehende Referenzliste verwendet hast (die du mir aber nicht geschickt hast), wo Sendernamen mit „&“ enthalten sind und auch noch einige Sender, die es in der LG-Senderliste gar nicht gibt.

Nur so kann ich mir erklären, weshalb da Programmnummer wie 1295 vergeben wurden, obwohl nur 1231 Sender in den LG-Senderlisten enthalten sind.

 

Das mit dem & bzw. &amp; ist ein Bug, der jetzt in der neuen Version behoben ist: https://github.com/PredatH0r/ChanSort/releases/tag/v2021-03-28_1804

Mit dieser Programmversion sollte deine alte Referenzliste wieder Funktionieren, um die Sender mit & korrekt einzureihen. Bitte verwende die neuere GlobalClone00001.txt aus deinem letzten zip NICHT weiter – dort steht das falsche &amp; drin.

 

Was „Welt der Wunder“, „LT1-OOE“ und „TV1-OOE“ betrifft bin ich noch am raten.

In einer ersten Datei waren die auf Plätzen ganz weit hinten, höher als die eigentliche Senderanzahl.

Bei deinen neueren Listen hast du sie wieder nach vorne gereiht auf 26 in der einen und 50 in der anderen.

Was macht dein Fernseher, wenn du die „20210324-2142….“-Liste ladest? 

Kannst du mit Eingabe von „50“ auf der Fernbedienung LT1-OOE ansehen? Mit Pr+/- ist er vermutlich nicht erreichbar, weil bei ihm „überspringen“ markiert ist.

Oder sind diese 3 Sender wirklich völlig verschwunden am Fernseher?

**Reply by ai-bits** [reply]
Hallo Horst,
no need to be sorry. Your support is better than practically any expensive commercial product I know.

I´m pretty sure I didn´t make too many blunders, but tested systematically. Here some reasons why the uploaded TLLs and templates are the way they are:

I think a template ending with 1254 (n-tv) stems from a standard presorted TLL for Austria. The raw Anderer Anbieter + Blindsuche TLL ends with 1079.
My first attempt was with the short- / 56-template and adding the Unsorted.
When that didn´t work out for The 3 (21 WdW, 26 LT1, 40 TV1) I removed The 3 from the 56-template and added them manually.
When that didn´t work I tried the 1254-template with and w/o The 3 and manual positioning for the latter.

With the 56-template and manual positioning The 3 were not Hidden in ChanSort for sure.
Sadly I didn´t check on the LG if they were there then, but hidden.

After the additional Manual scan I found The 3 and a handful of others in the same spot of the list in hidden state.

**Very suspiciously** one or two of The 3 were at a position I had had them in an on-TV-sorted list based on a pre-sorted Austria  list. Seemed like left-overs from there.

This is my most logical explanation I can provide.

So much for tonight. It´s getting very late, so I´ll rack my brains tomorrow if we can nail the reason and look at your input again for full coverage.
Cheers and thanks!
G.

**Comment by Master-Austria** [comment]
@PredatH0r -> I use an LG OLED55CX8LB and would like to give my feedback.
webOS 5.2.1-290911 (jhericurl-jardine)
Softwareversion 03.21.22
For searching channels I used following options -> automatic settings - cable - other providers - network search - cable tv, cable radio and analog cabel tv
then i exported the tv-list to a usb, copied it on the computer and changed everything with Chansort (ChanSort_2021-05-01). After the changes I imported the new list (after import tv rebooted automatically). Then again I imported the same list again and tada, the list was in the order like I changed it on the computer!

**Comment by Master-Austria** [comment]
@PredatH0r  -> I have an update regarding the LG OLED55CX8LB
After the last 2-3 TV firmware updates, my edited channel list was gone.
As soon as I switched a few channels up or down, the sorting was gone.
The channels were again in a completely different order, as was the numbering.
After the last update, I tried all possible variations of channel search and subsequent editing, all in vain.

Now, however, I seem to have found a solution, the channels are as I have set them and when I switch channels, everything remains as desired.
Current softwareversion installed on my TV:
webOS 5.2.3-290908 (jhericurl-jardine)
Softwareversion 03.23.15

I proceeded as follows:
On the TV: Settings -> Channels -> Channel search and settings -> Automatic setting -> Cable -> Other provider -> Complete incl. blind search (blind search no longer exists in network-mode) -> I did not activate the checkboxes "Search digital only" & "Skip encrypted channels".

After finishing the channel search, I copied the list to the stick as usual.
I then edited the list with ChanSort 2021-07-25 as follows.

I loaded the list and selected "Edit existing list" -> "Select action": Accept current channel order and then edit".

Then I switched the insert mode to "swap" and unchecked "Close gaps when moving/removing channels".
Then I put my channels in the desired order and by using the insert mode swap, the channels were swapped accordingly so that the original channel number (according to the cable provider) remained.
After my desired channels were in the desired order, I simply marked the remaining channels (also for DVB-C radio) and marked them with "Skip" and "Hide".
The TV restarted and the list remains as desired.

**Comment by PredatH0r** [comment]
Could you send me your list please so I can take a look at it?

DVB-C has always been a bit special and more difficult than DVB-S (like Astra 19.2E).
Most cable providers send out their recommended program numbers within the DVB-C data stream (so called "LCN" - logical channel numbers). 
DVB-S (or at least Astra 19.2E) doesn't do that, because every country/region/language prefers a different channel order. So for DVB-S the user can either run a blind search and reorder everything from scratch, or use a predefined list that's hardcoded in the TV's firmware and specific for each combination of satellite and country setting and impossible to reorder afterwards.

For DVB-T and DVB-C the TV often has a setting like "DTV auto update" or "automatic channel update" or similar, which would allow the  provider to automatically push out new numbers. If the TV uses a LCN based channel list, reordering with export/import typically doesn't work - because channels get reordered again as soon as the TV receives the cable provider's LCN again.

I'm curious to see if there are any LCN in your list. The part about swapping instead of assigning numbers freely is also intersting. I wonder if this is really a hard requirement. There shouldn't be a need for that if there are no LCNs, but it wouldn't be the first time LG surprises me in unpleasant ways.

**Comment by Master-Austria** [comment]
Sure, tomorrow I will send you the list! I will send you both, the original file after search and the final file re ordered with the swapping method.
Just for info, I turned off the option on my TV for the auto update of the channel list!

PS: if easier, feel free to write in German language


Von meinem iPhone gesendet



Von meinem iPhone gesendet

**Comment by Master-Austria** [comment]
Dear @PredatH0r 
in the appendix the files. One time the original GlobalClone (with above search options) and one time the changed channell list

[GlobalClone_bearbeitet.zip](https://github.com/PredatH0r/ChanSort/files/6905425/GlobalClone_bearbeitet.zip)
[GlobalClone_original.zip](https://github.com/PredatH0r/ChanSort/files/6905426/GlobalClone_original.zip)

**Comment by Master-Austria** [comment]
@PredatH0r a quick update from my side -> the list still works on my tv :)
The past few firmwares, I can't remember that the list lasted more then 2 days.
I really think the swapping instead of free assigning was the solution for my problem with WebOS 5. 

 @Info: my cable provider is Liwest-AT

**Comment by PredatH0r** [comment]
Funny note:
I have internet over TV cable, but not TV over cable. The same day I received your mail, I ran a channel search anyway and found all channels, interestingly with ORF 1 and 2 unencrypted.
And then i noticed ... the channels are in the exact same order as yours.
I'm also Liwest customer :)

Unfortunately my TV is an old LG 42LM620S with NetCast Firmware, so that doesn't really translate to WebOS.
But just like your list, even when selecting only neutral options, I get the preordered Liwest list, which is not ordered by frequency, but their logical channel numbers.
I haven't run any experiments yet how stable that list is when i randomly reorder it,
When I scan Astra 19.2E with neutral options, everything is ordered by frequency

**Comment by Joe2761** [comment]
Information about the function with LG webOS 6.4.0-1703:
webOS 6.4.0 has no parameters for setting in the channel manager.
The import and export works perfectly with chansort.
There was a problem because I had renamed the file name "GlobalClone00001.TLL" on the PC. It only works with the original name.

**Comment by reeno** [comment]
Worked flawlessly with a LG 55 Inch Class B3 series OLED 4K (OLED55B39LA).
- Auto Tuning > Tuning Mode "Cable" > Service Provider "Other Operators" > Cable settings "Network Search" > Tuning Options both "Digital Only Search" and "Skip Scramble Programmes" checked.
- Copy from TV to USB
- ChanSort
- Copy from USB to TV

Works!

**Comment by Mazda77** [comment]
Worked fine on LG 2024 TV (LG OLED C4). 
https://webos-forums.ru/post179680.html#p179680
Thank you

**Reply by ahmedawady** [reply]
please, how can I delete channels ?

**Reply by PredatH0r** [reply]
For webOS 5 ChanSort does not allow deleting channels.
The TV will respond unpredictably when doing so. It will insert the deleted channels at random places as soon as you tune in on another channel on the same transponder/frequency. To prevent this chaos, deleting is disabled and instead the channels that aren't assigned any numbers will be appended at the end of the list.

**Comment by NicerDicerPro** [comment]
LG65NANO917NA with webOs 5.6.1-6 (latest update as of today - tv shows "no updates").
Germany, Provider: Vodafone

- auto tune => mode "cable" => service provider "other operators" => network search => digital only and skip encrypted checked
- copy from tv to usb
- chansort
- copy from usb to tv
- senderlist is now broken (file gets imported, first channel is "Das Erste (HD)". Cant switch channels

.bak and sorted list attached.

ill try with a manual, blind search instead of network search now and see if it works.

It used to work last time, but i have no clue how i made it work since this was 4 years ago but vodafone decided to start changing frequencies so i have to retune.

[GlobalClone00001.zip](https://github.com/user-attachments/files/26420445/GlobalClone00001.zip)

Edit:

Second try also failed with same behaviour after importing

Created sender list with
auto tune => mode "cable" => service provider "other operators" => full search => blind search => digital only and skip encrypted checked

exported list tv to usb
chansort
imported from usb to tv
broken

Files for debugging:
[GlobalClone00001.zip](https://github.com/user-attachments/files/26420985/GlobalClone00001.zip)

I will just keep the auto assigned default list (which overlaps with what we want by 90%) and just wait for end of this year where we will move to a new house and finally switch to IP-TV.

Goodbye Vodafone forever.
